core/templates/dev/head/pages/exploration-player-page/services/exploration-recommendations.service.ts

```diff
@@ -46,8 +46,9 @@ oppia.factory('ExplorationRecommendationsService', [
           collection_id: null,
           include_system_recommendations: null
         };
-        if (GLOBALS.collectionId) {
-          recommendationsUrlParams.collection_id = GLOBALS.collectionId;
+        if (UrlService.getCollectionIdFromExplorationUrl()) {
+          recommendationsUrlParams.collection_id = (
+            UrlService.getCollectionIdFromExplorationUrl());
```

Comment:
Extract `UrlService.getCollectionIdFromExplorationUrl()` into variable and then sue that variable.

Reply:
Done.